Dockerfile

```diff
@@ -9,8 +9,37 @@ RUN apk add --no-cache npm \
   && npm i /src \
   && npm run build
 
-FROM nginx:latest
+################################################
 
-COPY --from=build  /src/build /usr/share/nginx/html
+FROM alpine:3 as lighttpd-builder
+ARG LIGHTTPD_VERSION="1.4.77"
+# inspired by source build description:https://git.lighttpd.net/lighttpd/lighttpd1.4/src/branch/master/INSTALL
 
-ENTRYPOINT ["/bin/bash", "-c", "/usr/share/nginx/html/generate_config.sh && nginx -g \"daemon off;\""]
+ADD https://download.lighttpd.net/lighttpd/releases-1.4.x/lighttpd-$LIGHTTPD_VERSION.tar.gz \
+  /tmp/lighttpd.tar.gz
+
+RUN apk add \
+  bsd-compat-headers \
+  build-base \
+  pcre2-dev \
+  scons \
+  && tar -C /tmp -xf /tmp/lighttpd.tar.gz \
+  && cd /tmp/lighttpd-$LIGHTTPD_VERSION \
+  && scons -j 4 build_fullstatic=1 build_dynamic=0
+
+RUN mkdir -p /rootfs/bin \
+      && cp /tmp/lighttpd-$LIGHTTPD_VERSION/sconsbuild/fullstatic/build/lighttpd /rootfs/bin/ && \
+    mkdir -p /rootfs/etc \
+      && echo "nogroup:*:10000:nobody" > /rootfs/etc/group \
+      && echo "nobody:*:10000:10000:::" > /rootfs/etc/passwd
+
+################################################
+
+FROM scratch
+
+COPY --from=lighttpd-builder --chown=10000:10000 /rootfs /
+COPY --from=node-builder /src/build /var/www/html/
```

Comment:
nit: we _might_ configure this e.g. to `/content` (or, maybe more opinionated: `/delivery-dashboard`), deviating from default (in containers, adhering to such deeply-nested directory-structures is not that much of a value-add 🤷🏻‍♂️ .